crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -519,12 +527,8 @@ impl Output for CliOutput {
             "Task ID".cell().bold(true),
```

Comment:
Maybe just "ID" to save some columns in smaller jobs?

Reply:
Good idea, I shortened it to just `ID`.